src/main/kotlin/io/github/gunpowder/commands/InvulnerableCommand.kt

```diff
@@ -27,25 +27,25 @@ package io.github.gunpowder.commands
 import com.mojang.brigadier.CommandDispatcher
 import com.mojang.brigadier.context.CommandContext
 import io.github.gunpowder.api.builders.Command
+import io.github.ladysnake.pal.AbilitySource
 import io.github.ladysnake.pal.Pal
 import io.github.ladysnake.pal.VanillaAbilities
 import net.minecraft.command.argument.EntityArgumentType
 import net.minecraft.server.command.ServerCommandSource
 import net.minecraft.server.network.ServerPlayerEntity
 import net.minecraft.text.LiteralText
 
-object GodCommand {
-    val ESSENTIALS_ABILITY_GOD = Pal.getAbilitySource("essentials", "godmode")
+object InvulnerableCommand {
+    private val ESSENTIALS_ABILITY_GOD: AbilitySource = Pal.getAbilitySource("essentials", "invulnerable")
 
     fun register(dispatcher: CommandDispatcher<ServerCommandSource>) {
         Command.builder(dispatcher) {
-            command("god") {
+            command("invulnerable", "godmode") {
```

Comment:
god was fine since it matches many other games, but either way at least add a short version of either.

Reply:
Like what?
I don't really like just adding too many aliases with similar names/characters when you can just auto-complete the name with like two words.